components/layout/mediaQuery/README.md

```diff
@@ -79,4 +79,21 @@ const LayoutMediaQueryBootstrap = LayoutMediaQueryFactory(BREAKPOINTS)
 </LayoutMediaQueryBootstrap>
 ```
 
-> **Find full description and more examples in the [demo page](https://sui-components.now.sh/workbench/layout/mediaQuery/demo).**
\ No newline at end of file
+### initialMediaQueries props
+
+This component accept and special prop call `initialMediaQueries`. This props is specially useful in SSR, because setting up you can force a media queries object, for environments where the `componentDidMount` never is called, like in Node.
```

Comment:
`This component accepts a special prop called initialMediaQueries. This prop is specially useful in SSR in order to force a initial media query object that will be used in environments where the componentDidMount is never called, like in Node.`